src/test/java/com/jcabi/aspects/ImmutableTest.java

```diff
@@ -89,6 +91,35 @@ public void catchesTypesMutableByClassInheritance() {
     }
 
     /**
+     * Test Version.
+     */
+    @Test
+    public void testVersion() {
+        try {
+            new Mutable();
+        } catch (final IllegalStateException ex) {
+            MatcherAssert.assertThat(
+                ex.getMessage(),
+                Matchers.not(
+                    Matchers.containsString(
+                        "${projectVersion}"
+                    )
+                )
+            );
+            MatcherAssert.assertThat(
+                ex.getMessage(),
+                Matchers.not(
+                    Matchers.containsString(
+                        "${buildNumber}"
+                    )
+                )
+            );
+            return;
+        }
+        MatcherAssert.assertThat("should not happen.", false);
```

Comment:
I have to catch the exception because I test exception's message.
Is it possible to do in other way?